src/components/views/spaces/threads-activity-centre/ThreadsActivityCentre.tsx

```diff
@@ -52,6 +53,15 @@ export function ThreadsActivityCentre({ displayButtonLabel }: ThreadsActivityCen
     const [open, setOpen] = useState(false);
     const roomsAndNotifications = useUnreadThreadRooms(open);
     const isReleaseAnnouncementOpen = useIsReleaseAnnouncementOpen("threadsActivityCentre");
+    const settingTACOnlyNotifs = useSettingValue<boolean>("Notifications.tac_only_notifications");
+
+    const emptyCaption = useCallback((): string => {
```

Comment:
The usage of `useCallback` seems too much for me in this use case.
A simple ternary/if is enough I think